packages/xstate-test/src/types.ts

```diff
@@ -125,7 +125,11 @@ export interface TestModelOptions<TState, TEvent extends EventObject>
     log: (msg: string) => void;
     error: (msg: string) => void;
   };
-  serializeTransition: (state: TState, event: TEvent | null) => string;
+  serializeTransition: (
+    state: TState,
+    event: TEvent | undefined,
+    prevState?: TState
```

Comment:
The mixed style of `| undefined` vs `?` in those serializers is a little bit weird. I can see some reasons why one could use something like this at times - but I don't understand this code well enough to know why this has been done like that here. Could you elaborate on why you have chosen this approach here?

Reply:
It used to be `| null`, should I change it back?

Reply:
Nah, either is fine - i just specifically wonder why `event` is a **required** (although it can be `undefined`) when the `prevState` is **optional**. Why the difference?